CONTRIBUTING.rst

```diff
@@ -6,9 +6,8 @@ Reporting bugs and asking questions
 
 You can post questions or issues or feedback through the following channels:
 
-1. `Github Discussions`_: For discussions about development, questions about usage, and feature requests.
-2. `GitHub Issues`_: For bug reports and feature requests.
-3. `StackOverflow`_
+1. `GitHub Issues`_: For bug reports and feature requests.
```

Comment:
Could you replace Github Discussions with Discourse Forum `For discussions about development and questions about usage.`?

Also line 21 needs to be updated as well.